routes/helpers/request-validators.js

```diff
@@ -34,6 +34,6 @@ class RequestValidators {
     }
 }
 
-RequestValidators.MAX_NUM_SECTIONS = 50;
+RequestValidators.MAX_NUM_SECTIONS = process.env.MAX_NUM_SECTIONS;
```

Comment:
This needs a default when the environment variable is not set (outside of a docker-compose) node 14+ allows a null coalesce operator `??` ie
```suggestion
RequestValidators.MAX_NUM_SECTIONS = process.env.MAX_NUM_SECTIONS ?? 50;
```

however this may need a simple if/else check